sidecar/internal/service/reclaimspace.go

```diff
@@ -52,6 +54,40 @@ func (rs *ReclaimSpaceServer) RegisterService(server grpc.ServiceRegistrar) {
 	proto.RegisterReclaimSpaceServer(server, rs)
 }
 
+// accessModeStrToInt convert access mode type string to int32.
+// Make sure to update this function as and when there are new modes introduced.
+func accessModeStrToInt(mode v1.PersistentVolumeAccessMode) csi.VolumeCapability_AccessMode_Mode {
```

Comment:
although the `..ToInt` name makes sense, you are returning a `csi.VolumeCapability_AccessMode_Mode` type. It might be clearer to rename the function to `k8sAccessModeToCSIAccessMode` or something like that

Reply:
possible to use https://github.com/kubernetes-csi/external-provisioner/blob/61e5b2d16c6bc7d3dbb0da81579cd57503f6c971/vendor/github.com/kubernetes-csi/csi-lib-utils/accessmodes/access_modes.go#L29?